screen-orientation/ios/Plugin/ScreenOrientation.swift

```diff
@@ -84,12 +88,13 @@ public class ScreenOrientation: NSObject {
         switch orientationType {
         case "any":
             return UIInterfaceOrientationMask.all
-        case "landscape", "landscape-primary":
+        case "landscape":
+            return UIInterfaceOrientationMask.landscape
+        case "landscape-primary":
+            return UIInterfaceOrientationMask.landscapeLeft
+        case "landscape-secondary":
             // UIInterfaceOrientationMask.landscapeRight is the same as UIDeviceOrientation.landscapeLeft
             return UIInterfaceOrientationMask.landscapeRight
```

Comment:
Must Fix: `landscape-secondary` must be `UIInterfaceOrientationMask.landscapeLeft`
Please add the CORRECT comment back.